tools/topology/topology2/include/components/pipeline.conf

```diff
@@ -84,6 +84,17 @@ Class.Widget."pipeline" {
 		token_ref	"sof_tkn_scheduler.word"
 	}
 
+	DefineAttribute."lp_mode" {
```

Comment:
@RanderWang can you confirm default value will be 0 (i.e. non lp mode) ?

Reply:
Each pipeline instance will set it. Now most of them set it to 0 except deep buffer.